src/API.jl

```diff
@@ -6,6 +6,7 @@ using UUIDs
 using Printf
 import Random
 using Dates
+using REPL
```

Comment:
Would be better to move the `levsort` somewhere common, but that may be beyond the scope of this PR.

Reply:
Several packages in the wild depend on `REPL.levsort` so moving it would break them.
THe ones i am aware of are **NamedColors.jl** and **DataFrames.jl**

Reply:
Maybe we should make it clear why we are depending on `REPL`
```suggestion
using REPL: levsort
```